cmake/Modules/FindMKLDNN.cmake

```diff
@@ -122,9 +122,6 @@ IF(NOT TARGET mkldnn)
   MESSAGE("Failed to include MKL-DNN target")
   RETURN()
 ENDIF(NOT TARGET mkldnn)
-IF(MKL_FOUND)
```

Comment:
@bddppq 
disabled this flag for MKLDNN, which caused the build failure.
This flag didn't impact on pytorch any more.